cypress/integration/dictionary/AddBulkConceptsToDictionaryPage/addBulkConceptsToDictionary.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Add bulk concepts to an existing dictionary
+  Background:
+    Given the user is logged in
+
+  @dictionary
+  Scenario: The user should be able to add bulk concepts from their preferred source
+    Given the user is on the view dictionary concepts page
+     When the user clicks the "Add concepts" button
+      And the user selects "Import existing concept"
+      And the user selects "Add bulk concepts"
+     Then the user should be on the "Add concepts in bulk from CIEL" page
+
+  @dictionary
+  Scenario: The user should be able to add a single bulk concept from their preferred source
```

Comment:
Remeber how to support the `addConceptsToDictionary` feature we had to add [this](https://github.com/openmrs/openmrs-ocl-client/blob/570ca1cbe0b603814eaaf227f35772f56ba838b6/cypress/support/step_definitions/dictionary/addConceptsToDictionary/addConceptsToDictionary.ts#L108) to ensure that the CIEL dictionary exists and had some concepts in it? We need to do the same thing here. It might be a good idea to migrate those before and after hooks to [the common file](https://github.com/openmrs/openmrs-ocl-client/blob/master/cypress/support/step_definitions/common/common.ts).